.github/workflows/test-and-publish-nightly.yml

```diff
@@ -36,10 +36,18 @@ jobs:
         container: ["public.ecr.aws/async-profiler/asprof-builder-x86:latest"]
         include:
           - runson:
-              display: macos-14-arm64
+              display: macos-13
```

Comment:
Why not including architecture in display name similarly to how it's done for linux?

Reply:
We could include it here, but we would be duplicating information which we have already in another variable (`matrix.architecture`). I'd personally like to remove `display` alltogether and only keep only `name` and `architecture`, what do you think?

Reply:
Sounds reasonable. Please go ahead and do the change. Thanks.

Reply:
Done